qa-tests-backend/src/test/groovy/ProcessVisualizationTest.groovy

```diff
@@ -85,6 +87,8 @@ class ProcessVisualizationTest extends BaseSpecification {
 
     @Tag("BAT")
     @Tag("RUNTIME")
+    // TODO(ROX-16461): Fails under AKS
```

Comment:
Is this TODO still open?

Reply:
Yes. And I think we even have a check for that.